src/parser/types.rs

```diff
@@ -182,12 +186,21 @@ pub enum Statement {
     ///   2 + 2;
     ///   return 0;
     /// }
-    Expr { expr: Expr, loc: Option<Location> },
+    Expr {
+        expr: Expr,
+        loc: Option<Location>,
+    },
+    //expr is like this in the parser, but later this does not work and returns an error
```

Comment:
I don't quite get this comment

Reply:
This is in reference to 
    //Standalone `Expr` statement, which means we can write programs like this:
    /// int main() {
    ///   2 + 2;
    ///   return 0;
not working because of the pest file, but we allow it in the parser